cdhweb/pages/management/commands/exodus.py

```diff
@@ -238,6 +285,32 @@ def form_pages(self):
         'root': Collection.objects.get(pk=get_root_collection_id())
     }
 
+    def profile_pages(self):
+        """Exodize all existing Profiles to new ProfilePage model."""
+        # all the fields on Profile that moved to Person will have been handled
+        # by django migrations; we just need to create new Page models
+        people_landing = PeopleLandingPage.objects.first()
+        if not people_landing:
+            return
+        for profile in Profile.objects.filter(user__person__isnull=False):
+            # NOTE not sure why Profile.user.person fails; works in console?
+            person = Person.objects.get(user=profile.user)
+            profile_page = ProfilePage(
+                person=person,
+                title=profile.title,
+                image=self.get_wagtail_image(profile.image) if profile.image else None,
+                education=profile.education,
+                bio=json.dumps([
+                    {"type": "migrated", "value": profile.bio},
+                ])
+            )
+            # added as child of people landing page so slugs are correct
```

Comment:
Where does slug get set?

Reply:
hmm. right now it would be getting generated automatically from the title, but we don't want that! I need to add a call to `convert_slug()` to use the old one, nice catch

Reply:
Yes! We need to make sure we use the same slugs that we have now

Reply:
added a call to `convert_slug()`